app/controllers/chargeback_controller.rb

```diff
@@ -254,6 +255,11 @@ def cb_rate_edit
                     id_measure = ChargebackRateDetailMeasure.find_by(:name => r[:measure]).id
                     detail.chargeback_rate_detail_measure_id = id_measure
                   end
+                  # Copy the currency id of the rate detail linking with the rate_detail_currency
+                  unless r[:type_currency].nil?
```

Comment:
I'd prefer `if r[:type_currency]` over a double negative unless you care about the different between false and nil.